client/src/App.js

```diff
@@ -1,19 +1,41 @@
 import React from "react";
 import { Routes, Route } from "react-router-dom";
 import HomePage from "./components/HomePage/HomePage";
+
+import NavBar from "./components/NavBar/NavBar";
+=======
```

Comment:
remove this

Reply:
You want me to remove navBar from app.js and put it in Homepage ? I have imported NavBar in app.js because footer was also included in app.js.

Reply:
I have removes those dashes.